### DISCUSSION TITLE
"Personal" vs "Business" Icons

### DISCUSSION THREAD

**Discussion by HartiganHM** [twilio-labs/paste#2382] [Request: Icon] [answered]
Hey all!

We are working on revamping a new form for the SendGrid Help Center. During this process, users have to choose whether they're using SG for a "Personal" or "Business" use case. For this, we'd like to have some new icons. I don't have a solid suggestion on what to use for "Personal", so any suggestions on your end would be appreciated!

Below is a rough mock-up of what the button selection will look like. Based on their selection we have them fill out a different type of form. Timeline wise, we would love to have these by end of May at the latest if possible. Let me know if you have any questions, thank you!

**Personal**
- Type: UI Icon
- [Streamline Icon 1](https://www.streamlinehq.com/icons/streamline-regular/files-folders/files-copyright/file-copyright-user)
- [Streamline Icon 2](https://www.streamlinehq.com/icons/streamline-regular/programing-apps-websites/apps/laptop-user)
- [Streamline Icon 3](https://www.streamlinehq.com/icons/streamline-regular/programing-apps-websites/apps-window/app-window-user)

**Business**
- Type: UI Icon
- [Streamline Icon 1](https://www.streamlinehq.com/icons/streamline-regular/building-construction/buildings/building-1)
- [Streamline Icon 2](https://www.streamlinehq.com/icons/streamline-regular/building-construction/buildings/buildings-1)
- [Streamline Icon 3](https://www.streamlinehq.com/icons/streamline-regular/building-construction/buildings/building-cloudy)

![image](https://user-images.githubusercontent.com/24458700/165999536-9af19c07-3ed9-4dab-a39c-9cd80dae2c67.png)

**Comment by shleewhite** [comment]
Hi @HartiganHM,

We would recommend using the Paste user icon for the personal use case
![Screen Shot 2022-05-03 at 1 24 22 PM](https://user-images.githubusercontent.com/8482334/166560039-4f9120ef-da0e-48ff-8cc0-b79ee3f8dcbb.png)

And for the business use case, we prefer the first option you posted: https://www.streamlinehq.com/icons/streamline-regular/building-construction/buildings/building-1. 

For timeline, we are currently focusing on reliability and not releasing new features - you can see more information here: https://github.com/twilio-labs/paste/discussions/2370. 

Please let me know if you have any additional questions!

**Reply by HartiganHM** [reply]
Hey, @shleewhite! Thanks for the follow-up. In regards to not releasing new features, I assume this icon would fall into that bucket, so can you and the team suggest an alternative for icon use? Should we use a custom SVG, etc.?

Thanks!

**Reply by shleewhite** [reply]
You can definitely use a custom SVG for now. I also just made a ticket for my team so that when we do feature work again, we can prioritize this icon: https://issues.corp.twilio.com/browse/DSYS-3332. We will follow up on this thread in the future when this is released.

**Reply by HartiganHM** [reply]
@shleewhite Do we have a Twilio account for Streamline? It looks like the SVG of the "Business" icon is locked. Either that or could someone on the Paste side acquire the file for me?

**Reply by nkrantz** [reply]
@HartiganHM we have a Twilio account through brand, I'm figuring out how you can get access to that. In the meantime, @loreina is going to get the file for you!

**Reply by HartiganHM** [reply]
Y'all are my heroes ❤️

**Comment by loreina** [comment]
Hey @HartiganHM, we'll be able to add this icon to Figma but it won't be available in Paste for a while as we're focused on resiliency work right now. I'll add the icon to Figma (might be a couple days) and reach out to you over Slack to share the svg file. Thanks!

**Comment by SiTaggart** [comment, marked as answer]
Hi @HartiganHM 

The Business icon is now available in Figma and NPM https://paste.twilio.design/components/icons
